dashboard/lib/service/google_authentication.dart

```diff
@@ -26,7 +28,7 @@ class GoogleSignInService extends ChangeNotifier {
     });
 
     try {
-      _googleSignIn.signInSilently();
+      unawaited(_googleSignIn.signInSilently());
```

Comment:
wait, why is there a try/catch that doesn't actually catch anything?

Reply:
That is a good question, for another time.